### PR TITLE
Change simple code in book

### PR DESCRIPTION
**Pull request by ccqpein**
In `Get started with Druid` of book, the simple code:

```rust
fn main() -> Result<(), PlatformError> {
    AppLauncher::with_window(WindowDesc::new(build_ui)).launch(())?;
    Ok(())
}
```

doesn't pass the compiler.  Change it to:

```rust
fn main() -> Result<(), PlatformError> {
    AppLauncher::with_window(WindowDesc::new(build_ui())).launch(())?;
    Ok(())
}
```

will work

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -58,6 +58,8 @@ You can find its changes [documented below](#070---2021-01-01).
 
 ### Docs
 
+- Fix example code in `Get started with Druid` chapter of book ([#1698] by [@ccqpein])
```
Comment (SecondFlight) [inline CHANGELOG.md:61]: You have to add to the list of links at the bottom of the file for the `[#1698]` and `[@ccqpein]` shortcuts to work

Comment (ccqpein) [reply CHANGELOG.md:61]: Thanks for notifying me, let me change it now.

Comment (ccqpein) [reply CHANGELOG.md:61]: Just done. Please check it out.

**Review by SecondFlight — APPROVED** [review]

**Comment by SecondFlight** [comment]
I'll merge this tomorrow morning if there are no other comments.

**Comment by memoryruins** [comment]
This change causes the example code to no longer compile with `0.7`, although the book instructs the user to use `0.7` right above the example. New users have ran into this on discord.

I realize users of master branch can run into this without this change #1637, but the book doesn't currently specify using a git version. Should this be reverted?

**Comment by SecondFlight** [comment]
Based on [this Zulip conversation](https://xi.zulipchat.com/#narrow/stream/147926-druid/topic/druid.231698), I've opened a PR (#1732) to update the book to use the latest git version. This was an oversight when I reviewed this PR. Sorry for the confusion it caused!

**Pull request merged**